app/js/views/home.js

```diff
@@ -2,15 +2,35 @@ var app = app || {};
 
 app.views = app.views || {};
 
+
 app.views.Home = (function() {
 
 	'use strict';
 
 	return Backbone.View.extend({
 
+		collection: app.trees,
 		className: 'home',
 		template: '#template-home',
 
+		getTrees: function(cb) {
+
+			var self = this;
+			var treesArray = [];
+			this.collection.fetch({
+				success: function(response) {
+					_.each(response.toJSON(), function(item) {
+						console.log('Successfully got tree with id: ' + item.id);
+					})
+					treesArray = response.toJSON();
+					cb(treesArray);
+				},
+				error: function() {
+					console.log('Failed to get trees!');
```

Comment:
Instead of using `console.log`, should execute the callback function with an error like this:
```js
cb(new Error('Failed to get trees!'));
```